test/fit_tests/README.md

```diff
@@ -0,0 +1,123 @@
+
+# Running FIT
+
+## Requirements and Setup
+
+FIT tests are intended to be run on Ubuntu 14+ Linux.
+Test harness may be run on appliance host (localhost), or third party machine.
+Deployment scripts must be run under third party Ubuntu Linux host.
+Tests require the following virtual environment commands be executed:
+
+    virtualenv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+
+
+## Organization
+
+- 'tests' is the test 'harness'
+- 'common' contains any common library functions
+- 'deploy' contains deployment and installation scripts
+- 'templates' contains script templates for making new tests
+- 'util' contains non-test utilities
+
+## Configuration
+
+Local runtime parameters are set from the 'config/global_config.json' file.
+Stack definitions are set from the 'config/stack_config.json' file.
+An alternate configuration directory can be selected using the -config argument.
+More details in config/README.mg.
+
+## Running the tests
+
+All FIT tests can be run from the wrapper 'run_tests.py':
+
+    usage: run_tests.py [-h] [-test TEST] [-config CONFIG] [-group GROUP]
+                        [-stack STACK] [-ora ORA] [-version VERSION] [-xunit]
+                        [-list] [-sku SKU] [-obmmac OBMMAC | -nodeid NODEID]
+                        [-http | -https] [-v V]
+
+    Command Help
+
+    optional arguments:
+      -h, --help        show this help message and exit
+      -test TEST        test to execute, default: tests/
+      -config CONFIG    config file location, default: fit_tests/config
+      -group GROUP      test group to execute: 'smoke', 'regression', 'extended',
+                        default: 'all'
+      -stack STACK      stack label (test bed), overrides -ora
+      -ora ORA          OnRack/RackHD appliance IP address or hostname, default:
+                        localhost
+      -version VERSION  OnRack version, example:onrack-release-0.3.0, default:
+                        onrack-devel
+      -xunit            generates xUnit XML report files
+      -list             generates test list only
+      -sku SKU          node SKU, example:Phoenix, default=all
+      -obmmac OBMMAC    node OBM MAC address, example:00:1e:67:b1:d5:64,
+                        default=all
+      -nodeid NODEID    node identifier string of a discovered node, example:
+                        56ddcf9a8eff16614e79ec74
+      -http             force using http protocol
+      -https            force using https protocol
+      -v V              Verbosity level of console output, default=0, Built Ins:
+                        0: No debug, 2: User script output, 4: rest calls and
+                        status info, 6: other common calls (ipmi, ssh), 9: all the
+                        rest
+
+This example will run the RackHD installer onto stack 1 via the wrapper script:
+
+    python run_tests.py -stack 1 -test deploy/run_rackhd_installer.py
```

Comment:
In the instance of http://localhost:9090, I suggest that you specify 9090 as the http port in the configuration file, then use 172.31.128.1 as the ora address and for http. This will bypass the default. Same goes for https.